RELEASE/scripts/autoscend/quests/level_10.ash

```diff
@@ -15,10 +15,16 @@ boolean L10_plantThatBean()
 		set_property("questL10Garbage", "step1");
 		return true;
 	}
-	if(item_amount($item[Enchanted Bean]) > 0)
+	if(item_amount($item[Enchanted Bean]) > 0 && auto_is_valid($item[Enchanted Bean]))
 	{
 		use(1, $item[Enchanted Bean]);
 		return true;
+	} else if(item_amount($item[Enchanted Bean]) > 0)
+	{ 
+		//can't use the bean, but we can visit coffee grounds directly to plant it
```

Comment:
```suggestion
	if(item_amount($item[Enchanted Bean]) > 0)
	{ 
```
Just have all paths use the visit_url as it'll work for everything (or at least all the currently existing paths, if some future path needs special handling like Ed does we can add support then).